homeassistant/components/template/binary_sensor.py

```diff
@@ -417,8 +417,8 @@ def _handle_coordinator_update(self) -> None:
             self.async_write_ha_state()
             return
 
-        # state without delay. None means rendering failed.
-        if self._attr_is_on == state or state is None or delay is None:
+        # state without delay.
+        if self._attr_is_on == state or delay is None:
```

Comment:
From my undestanding, the comment was misleading.
If rendering failed, it would have been caught by the availability.

Happy to undo this code change if needed.